src/energy_and_flux.cpp

```diff
@@ -58,7 +58,7 @@ double fields::field_energy_in_box(const volume &where) {
   return electric_energy_in_box(where) + cur_step_magnetic_energy;
 }
 
-static complex<double> dot_integrand(const complex<double> *fields, const vec &loc, void *data_) {
+static complex<realnum> dot_integrand(const complex<realnum> *fields, const vec &loc, void *data_) {
```

Comment:
Why is the return value changed?

Reply:
Return type reverted to `std::complex<double>`.